lxd/storage/utils.go

```diff
@@ -1377,3 +1377,13 @@ func ValidVolumeName(volumeName string) error {
 
 	return nil
 }
+
+// GetPoolDefaultBlockSize returns the default block size for the specified storage pool according to its driver.
+func GetPoolDefaultBlockSize(s *state.State, poolName string) (string, error) {
```

Comment:
This isn't quite the same as `DefaultVMBlockFilesystemSize` as it has a state argument.
Why is that?
Can we not use the same pattern as `DefaultVMBlockFilesystemSize`?

Reply:
We can use the same pattern as `DefaultVMBlockFilesystemSize`, but `GetPoolDefaultBlockSize` aims to get the default block size using the pool name, not the driver name. So we need to get the pool data from the database to know what its driver is. And this is like this because this is what we need for `instance_placement`.

Alternatively, I can create a function like `DefaultVMBlockFilesystemSize` and get the pool info before calling it. Hope this was clear.

Reply:
understood thanks, this is fine